packages/nimble-components/src/table-column/base/models/column-internals.ts

```diff
@@ -153,6 +157,12 @@ export class ColumnInternals<
     @observable
     public hideHeaderIndicators = false;
 
+    /**
+     * How to align the header content.
+     */
+    @observable
+    public headerAlignment?: TableColumnAlignment;
```

Comment:
instead of making this undefined can this just be `TableColumnAlignment.left`? That seems to be the fallback being used below when undefined